alpaca_trade_api/rest.py

```diff
@@ -891,13 +878,11 @@ def get_latest_crypto_trade(self, symbol: str, exchange: str) -> TradeV2:
             api_version='v1beta1')
         return self.response_wrapper(resp['trade'], TradeV2)
 
-    def get_latest_crypto_trades(self,
-                                 symbols: List[str],
-                                 exchange: str) -> LatestTradesV2:
+    def get_latest_crypto_trades(self, symbols: List[str]) -> LatestTradesV2:
         resp = self.data_get(
-            '/crypto/trades/latest',
-            data={'exchange': exchange, 'symbols': _join_with_commas(symbols)},
-            api_version='v1beta1')
+            '/crypto/latest/trades',
+            data={'symbols': _join_with_commas(symbols)},
+            api_version='v1beta2')
         return self.response_wrapper(resp['trades'], LatestTradesV2)
 
     def get_latest_crypto_quote(self, symbol: str, exchange: str) -> QuoteV2:
```

Comment:
Let's remove the `exchange` parameter from all crypto methods. `exchange` parameter is no longer available on `v1beta2` endpoints.

Reply:
Okay so the functions that still had exchanges were v1beta1 so I removed them. They aren't needed anyway, unless you want separate single-symbol and multi-symbol for each data type (bars, multibars, latest multibars, latest bars).